treeherder/perf/management/commands/backfill_perf_jobs.py

```diff
@@ -1,5 +1,7 @@
+from django.conf import settings
 from django.core.management.base import BaseCommand
 
+from treeherder.perf.investigation_aids import DEFAULT_ROOT_URL as root_url
```

Comment:
Treeherder's `repository` has a `tc_root_url` field. I think it would be better to use that, than to hard-code the URL.